### PR TITLE
feat: meditation card and some bug fixes

### LINKED ISSUE #325: resources are duplicated for no reason
**Describe the bug**
Resources are being duplicated for no reason
cards created In layouts are not being used in the content_main, rather being copied and pasted inside content_main.xml.

**Would you like to work on the issue?**
Will fix it with the new meditation card

### PR DESCRIPTION
**Pull request by adityastic**
Fixes #325 #321

**Changes**: 
- Added Meditation card and removed duplicate layouts

**Screenshot/s for the changes**:

### CODE REVIEW

File: app/src/main/res/layout/card_meditation.xml [hunk 1]
```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.v7.widget.CardView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:id="@+id/meditation_card"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    app:cardCornerRadius="12.0dip"
```
Comment (CloudyPadmal) [inline app/src/main/res/layout/card_meditation.xml]: Shouldn't these better be `12dp`?

File: app/src/main/res/layout/card_meditation.xml [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+            android:layout_height="wrap_content"
+            android:layout_centerInParent="true"
+            android:orientation="horizontal"
+            android:padding="16.0dip">
+
+            <ImageView
+                android:layout_width="match_parent"
+                android:layout_height="match_parent"
+                android:layout_marginStart="8dp"
+                android:layout_marginEnd="8dp"
```
Comment (CloudyPadmal) [inline app/src/main/res/layout/card_meditation.xml]: Let's add these hardcoded dimensions to `dimens.xml`

File: app/src/main/res/layout/card_meditation.xml [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+                    android:text="Meditation"
+                    android:textAppearance="@style/TextAppearance.AppCompat.Display1"
+                    android:textColor="#fff" />
+
+                <android.support.v7.widget.AppCompatTextView
+                    android:id="@+id/meditation_subtitle"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:padding="12.0dip"
+                    android:text="to (cause someone to) become less active and more calm and happy"
```
Comment (CloudyPadmal) [inline app/src/main/res/layout/card_meditation.xml]: Similarly these hardcoded strings to `strings.xml`

**Review by CloudyPadmal — CHANGES_REQUESTED** [review]
Excellent work @adityastic . Thank you for the PR. There are some little things need to get fixed.

**Comment by CloudyPadmal** [comment]
@adityastic what's your idea about the gap between two cards. It seems it would look nicer if they are closer

**Comment by adityastic** [comment]
> @adityastic what's your idea about the gap between two cards. It seems it would look nicer if they are closer

true, it should match the left and right padding. Let me fix all of it

File: app/src/main/res/layout/card_meditation.xml [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+            android:orientation="horizontal"
+            android:padding="16.0dip">
+
+            <ImageView
+                android:layout_width="match_parent"
+                android:layout_height="match_parent"
+                android:layout_marginStart="8dp"
+                android:layout_marginEnd="8dp"
+                android:layout_weight="1.9"
+                android:src="@drawable/ic_card_meditation"
```
Comment (CloudyPadmal) [inline app/src/main/res/layout/card_meditation.xml]: `android:src` --> `app:srcCompat`

**Review by CloudyPadmal — CHANGES_REQUESTED** [review]

File: app/src/main/res/layout/card_meditation.xml (hunk 3 above)
Comment (jddeep) [inline app/src/main/res/layout/card_meditation.xml]: @adityastic The text labels and descriptions are already present in `strings.xml`. Please use from there,

**Review by jddeep — CHANGES_REQUESTED** [review]

File: app/src/main/res/layout/card_meditation.xml (hunk 3 above)
Comment (adityastic) [reply app/src/main/res/layout/card_meditation.xml]: great

**Comment by adityastic** [comment]
@CloudyPadmal @jddeep updated

**Review by CloudyPadmal — APPROVED** [review]

**Review by suresh-gandhi — APPROVED** [review]

**Review by jddeep — APPROVED** [review]

**Pull request merged**